client/cypress/e2e/allocationRewardsBox.cy.ts

```diff
@@ -0,0 +1,491 @@
+
+        cy.get('[data-test=AllocationRewardsBox__Slider__thumb]').then(sliderButtonEl => {
+          const sliderButtonDimensions = sliderButtonEl[0].getBoundingClientRect();
+
+          // track 0 is hidden under the thumb
+          cy.get('[data-test=AllocationRewardsBox__Slider__track--0]').should(
+            'have.css',
+            'width',
+            `${sliderButtonDimensions.width}px`,
+          );
+          cy.get('[data-test=AllocationRewardsBox__Slider__track--1]').should(
+            'have.css',
+            'width',
+            `${sliderElWidth}px`,
+          );
+
+          const pageX0 = sliderButtonDimensions.left;
+          const pageX50 =
+            sliderButtonDimensions.left - sliderButtonDimensions.width / 2 + sliderElWidth / 2;
+          const pageX100 =
+            sliderButtonDimensions.left - sliderButtonDimensions.width / 2 + sliderElWidth;
+
+          cy.get('[data-test=AllocationRewardsBox__Slider__thumb]')
+            .trigger('mousedown', {
+              pageX: pageX0,
+            })
+            .trigger('mousemove', {
+              pageX: pageX50,
+            })
+            .trigger('mouseup', {
+              pageX: pageX50,
+            });
+
+          cy.get('[data-test=AllocationRewardsBox__Slider__track--0]').should(
+            'have.css',
+            'width',
+            `${(sliderButtonDimensions.width + sliderElWidth) / 2}px`,
+          );
+          cy.get('[data-test=AllocationRewardsBox__Slider__track--0]').should(
+            'have.css',
+            'background-color',
+            'rgb(45, 155, 135)',
+          );
+          cy.get('[data-test=AllocationRewardsBox__Slider__track--1]').should(
+            'have.css',
+            'width',
+            `${(sliderButtonDimensions.width + sliderElWidth) / 2}px`,
+          );
+          cy.get('[data-test=AllocationRewardsBox__Slider__track--1]').should(
+            'have.css',
+            'background-color',
+            'rgb(255, 150, 1)',
+          );
+
+          cy.get('[data-test=AllocationRewardsBox__section__value--0]')
+            .invoke('text')
+            .should('eq', '5 GWEI');
+          cy.get('[data-test=AllocationRewardsBox__section__value--1]')
+            .invoke('text')
+            .should('eq', '5 GWEI');
+
+          cy.get('[data-test=AllocationRewardsBox__Slider__thumb]')
+            .trigger('mousedown', {
+              pageX: pageX50,
+            })
+            .trigger('mousemove', {
+              pageX: pageX100,
+            })
+            .trigger('mouseup', {
+              pageX: pageX100,
+            });
+
+          cy.get('[data-test=AllocationRewardsBox__Slider__track--0]').should(
+            'have.css',
+            'width',
+            `${sliderElWidth}px`,
+          );
+          cy.get('[data-test=AllocationRewardsBox__Slider__track--0]').should(
+            'have.css',
+            'background-color',
+            'rgb(45, 155, 135)',
```

Comment:
As discussed, please use chai-colors that we already have in `projects.cy.ts` and assert HEX everywhere.